feature-libs/customer-ticketing/styles/components/_customer-ticketing-dialog.scss

```diff
@@ -38,6 +39,13 @@
         .cx-customer-ticket-input-hint {
           font-size: 0.75rem;
           font-style: italic;
+          text-align: end;
+          margin-bottom: 0;
+        }
+        .cx-customer-ticket-file-upload-hint {
```

Comment:
I believe we can remove `.cx-customer-ticket-input-hint` that was done in Parth's PR as I noticed in the wireframes that they're all aligned at the start and not at the end. Therefore, update the class in the reopen component as well please! 
![Screen Shot 2022-09-01 at 4 27 35 PM](https://user-images.githubusercontent.com/29099466/188006025-d6bcbaeb-5084-4e00-8c6a-859a6ac9696c.png)

Reply:
Have made changes in reopen component to use cx-customer-ticket-file-upload-hint class. We require '.cx-customer-ticket-input-hint' as well for 'Characters left' text at the bottom of message.